tmc-langs-framework/src/test/java/fi/helsinki/cs/tmc/langs/ExerciseBuilderTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package fi.helsinki.cs.tmc.langs;
+
+import fi.helsinki.cs.tmc.langs.ExerciseBuilder;
+import com.google.common.io.Files;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ * @author villheik
+ */
+public class ExerciseBuilderTest {
+
+    public ExerciseBuilderTest() {
+    }
+
+    private File createTemporaryCopyOf(File file) throws IOException {
+        File tempFolder = Files.createTempDir();
+        FileUtils.copyDirectory(file, tempFolder);
+        return tempFolder;
+    }
```

Comment:
Keep a list of these directories and delete them in a `@TearDown` method. Currently this leaves junk in `/tmp`.